daemon/command/config_unix.go

```diff
@@ -27,6 +27,7 @@ func installConfigFlags(conf *config.Config, flags *pflag.FlagSet) {
 	flags.BoolVar(&conf.BridgeConfig.DisableFilterForwardDrop, "ip-forward-no-drop", false, "Do not set the filter-FORWARD policy to DROP when enabling IP forwarding")
 	flags.BoolVar(&conf.BridgeConfig.EnableIPMasq, "ip-masq", true, "Enable IP masquerading for the default bridge network")
 	flags.BoolVar(&conf.BridgeConfig.EnableIPv6, "ipv6", false, "Enable IPv6 networking for the default bridge network")
+	flags.Var(opts.NewNamedMapOpts("bridge-nftables-priorities", conf.NftablesPriorities, nil), "bridge-nftables-priority", "Base chain priorities for bridge driver nftables")
```

Comment:
Can you make this explicitly use `conf.BridgeConfig.NftablesPriorities` ? Makes it clearer it's related to the bridge only.